src/js/modules/controller/taskcontroller.js

```diff
@@ -0,0 +1,157 @@
+import Task from '../model/task.js';
+
+class TaskController {
+  static taskList = document.querySelector('#tasks-list');
+
+  static createTag = (tagName, textContent = null, className = null) => {
+    const tag = document.createElement(tagName);
+    tag.textContent = textContent;
+    tag.className = className;
+    return tag;
+  };
+
+  static createTaskRow = (task) => {
+    const taskRow = TaskController.createTag('li', null, 'task-item');
+    const taskForm = TaskController.createTag('form', null, 'task-form');
+    const descriptionElement = TaskController.createTag(
+      'input',
+      null,
+      'item description',
+    );
+    descriptionElement.type = 'text';
+    descriptionElement.name = 'description';
+    const indexElement = TaskController.createTag('div', null, 'item index');
+    const completedElement = TaskController.createTag('div', null, 'item completed');
+
+    const dragNdrop = TaskController.createTag(
+      'span',
+      null,
+      'fas fa-ellipsis-v vertical-3-dot',
+    );
+    const deleteIcon = TaskController.createTag(
+      'span',
+      null,
+      'fa fa-trash-o delete-icon',
+    );
+
+    const checkboxElement = TaskController.createTag('input', null, 'check-input');
+    checkboxElement.type = 'checkbox';
+    checkboxElement.name = 'completed';
+
+    checkboxElement.checked = task.completed;
+    /* checkboxElement.addEventListener("change", (event) => {
+      TaskController.resetForRemove();
+      const newTask = new Task(task.index, task.description, task.completed);
+      console.log('**********************')
+      console.log(newTask)
+      if(newTask){
+        newTask.setCompleted(checkboxElement.checked);
+        console.log(newTask)
+      }
+
+    }); */
```

Comment:
Good job! but it's highly recommended to keep your repository clean. So please make sure to remove it.

Reply:
OK. Thanks

Reply:
Fixed.